spec/cc/cli/init_spec.rb

```diff
@@ -170,11 +171,12 @@ module CC::CLI
           YAML.safe_load(new_content).must_equal({
             "engines" => {
               "csslint" => { "enabled"=>true },
+              "duplication" => { "enabled" => true, "config" => { "languages" => ["ruby", "javascript", "python", "php"] } },
```

Comment:
Looks like the array format is still [valid](https://github.com/codeclimate/codeclimate-duplication/blob/ce27d9e3b99513115968df64eafa13c8693d5a3c/lib/cc/engine/analyzers/engine_config.rb#L57-L67). The hash format is necessary when you're overriding mass threshold for a language.